pyproject.toml

```diff
@@ -1,40 +1,43 @@
+[build-system]
+requires      = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
 [tool.poetry]
 name = "darglint2"
 version = "1.8.2"
-description = "A utility for ensuring Google-style docstrings stay up to date with the source code."
+description = "A utility for ensuring docstrings stay up to date with the source code."
 authors = ["terrencepreilly <terrencepreilly@gmail.com>"]
+maintainers = [
+    "Antti Kaihola <13725+akaihola@users.noreply.github.com>",
+    "yfprojects <real-yfprojects@users.noreply.github.com>",
+]
 license = "MIT"
+repository = "http://github.com/akaihola/darglint2"
+documentation = "https://raw.githubusercontent.com/akaihola/darglint2/master/docs/poster.pdf"
 keywords = ["documentation", "linter", "development"]
 classifiers = [
     'Intended Audience :: Developers',
     'Topic :: Software Development :: Documentation',
     'Topic :: Software Development :: Quality Assurance',
     'License :: OSI Approved :: MIT License',
-    'Programming Language :: Python :: 3.9',
-    'Programming Language :: Python :: 3.8',
-    'Programming Language :: Python :: 3.7',
-    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3 :: Only',
```

Comment:
I think it would be good to explicitly list the supported Python versions here so they show up on the PyPI page.